Makefile

```diff
@@ -1,9 +1,9 @@
 OPENRESTY_PREFIX=/usr/local/openresty
 
-LUA_VERSION := 5.1
+#LUA_VERSION := 5.1
 PREFIX ?=          /usr/local
 LUA_INCLUDE_DIR ?= $(PREFIX)/include
-LUA_LIB_DIR ?=     $(PREFIX)/share/lua/$(LUA_VERSION)
+LUA_LIB_DIR ?=     $(PREFIX)/lib/lua/$(LUA_VERSION)
```

Comment:
Let's keep the original LUA_LIB_DIR and comment out the LUA_VERSION.
Instead we add an echo warning message under the install target.

Reply:
good idea